### DISCUSSION TITLE
第三节课笔记（23班）

### DISCUSSION THREAD

**Discussion by wangyijunlyy** [InternLM/Tutorial#417] [笔记]
笔记提交

**Comment by i-square** [comment]
https://i-square.github.io/p/InternLM-tutorial-campsection3-Build-Knowledge-Base-Using-InternLM-and-LangChain/

**Comment by yaoyaotaba** [comment]
https://www.zhihu.com/question/609483833/answer/3361386339

**Comment by PlateAss** [comment]
https://plateass.github.io/class3

**Comment by Epi10gue** [comment]
http://t.csdnimg.cn/c0LyW

**Comment by YangZhC** [comment]
https://blog.csdn.net/donotpython/article/details/135761538
